chainer/functions/array/concat.py

```diff
@@ -40,9 +41,22 @@ def check_type_forward(self, in_types):
                 type_check.expect(in_types[0].shape[d] == in_types[i].shape[d])
 
     def forward(self, xs):
+        if (intel64.should_use_ideep('>=auto')
+                and intel64.inputs_all_ready(xs, (4,))):
+
```

Comment:
A minor but should we remove this empty line?

Reply:
Fixed (added comment instead)